html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Coding Form</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>
+      <form action="" method="get" class="coding-form">
+        <div>
+          <div><label for="email">Email</label></div>
+          <input type="text" name="email" id="email" required/>
+        </div>
+        <div>
+          <label for="options">Why is coding cool?</label>
+          <div>
+            <select name="options" id="options">
+              <option value="Select an option">Select an option</option>
+              <option value="Because it's interesting">Because it's interesting</option>
+              <option value="Because it's fun">Becuase it's fun</option>
+              <option value="Because it's powerful">Because it's powerful</option>
+            </select>
+          </div>
+        </div>
+        <br>
+        <label for="radio">Favorite Language (choose one)?</label>
+        </br>
+        <div>
+          <input type="radio" id="html" name="language" value="HTML"/>
+          <label for="html">HTML</label>
+          <input type="radio" id="css" name="language" value="CSS"/>
+          <label for="css">CSS</label>
+          <input type="radio" id="javascript" name="language" value="JavaScript"/>
+          <label for="javascript">JavaScript</label>
+        </div>
+        <p>
+          Would you like to sign up for more information?
+        </p>
+        <input type="checkbox" id="sign-me-up" name="sign-me-up" value="Sign me up!">
+        <label for="sign-me-up"> Sign me up!</label>
+        <div>
+          <label for="comments">Comments</label>
+          <br>
+          <textarea name="comments" cols="30" rows="10"></textarea>
```

Comment:
don't forget the `id` on the text area which matches the `for` attribute on the label